backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -321,4 +348,26 @@ Java_backtraceio_library_base_BacktraceBase_dumpWithoutCrash__Ljava_lang_String_
                                                                                    jboolean set_main_thread_as_faulting_thread) {
     DumpWithoutCrash(message, set_main_thread_as_faulting_thread);
 }
+
+bool EnableClientSideUnwinding() {
```

Comment:
@konraddysput here is the native function you can use to enable client-side unwinding from backtrace-unity